_episodes/15-practices.md

```diff
@@ -228,6 +185,71 @@ Peak rule.
     While it has been criticized for not strongly predicting what's remembered,
     it's always worth trying to end a lesson on a high note.
 
+Instructor guides.
+:   Many of the Software and Data Carpentry lessons have an 
+    instructor's guide, with notes 
+    from instructors who have already taught the material.  This can be a valuable 
+    resource when preparing lessons, especially when teaching a lesson for the first time.  
+    The Software Carpentry instructor guides are linked on each lesson page; the 
+    instructor guides for Data Carpentry lessons are on 
+    their [main lesson page][here]({{ site.dc_site }}/lessons/).  
```

Comment:
Two times text in square brackets, I suspect it should be 

`main lesson page [here]({{ site.dc_site }}/lessons/).`

Also, not all lessons listed on the DC lessons page have links to the instructor guides...